miracl/stats/miracl_stats_ace_parser.py

```diff
@@ -27,7 +29,7 @@ def parsefn(self) -> argparse.ArgumentParser:
                 "CONTROL_VOXILIZED_SEGMENTED_TIF_EXAMPLE_PATH",
             ),
             help="FIRST: path to base control directory.\nSECOND: example path to control subject voxelized tif file (voxelized_seg_*.nii.gz)",
-            nargs=2,
+            nargs="+",
         )
 
         single_multi_args_group.add_argument(
```

Comment:
How elaborate would it be to change the `-e` flag to `-t` and the args from "EXPERIMENT_BASE_DIR EXPERIMENT_TIFF_DIR_EXAMPLE" to "TREATED_BASE_DIR TREATED_TIFF_DIR_EXAMPLE" as well as changing mentions of "experiment" in the help to "treated"?

Reply:
I will look into how many occurrences there are and how hard it will be

Reply:
Thanks - @AAttarpour prefers "treated" over "experimental" so let's hope it's not too much work to change all occurrences!